distribution/src/main/resources/docker-compose/docker-compose.yml

```diff
@@ -319,8 +332,9 @@ services:
         limits:
           memory: 256m
     environment:
-      SERVICES: s3
+      SERVICES: s3,sqs
```

Comment:
I don't think the messages are going to come via SQS. It's not been defined yet, but we might need to poll on a REST API. I have no problem with this being the first version of our code though.

Reply:
I'm aware of that, but for now lets leave it as it is, as there's not much we can relay on